src/app/core/store/organizations/organizations.reducer.ts

```diff
@@ -28,11 +28,12 @@ export function organizationsReducer(state: OrganizationsState = initialOrganiza
     case OrganizationsActionType.CREATE_SUCCESS:
       return groupsAdapter.addOne(action.payload.organization, state);
     case OrganizationsActionType.UPDATE_SUCCESS:
-      return organizationsAdapter.updateOne({id: action.payload.organization.code, changes: action.payload.organization}, state);
+      console.log('reducer', action.payload);
```

Comment:
One of the console logs you have mentioned? :)

Reply:
Haha, yes :)